### PR TITLE
Supports ICS , Dynamic ICS, Reset Button

### PR DESCRIPTION
**Pull request by calawhkbu**
- Export One time ICS (Current LoggedIn User with all Tickets and Projects)
 - Auto Generate Event.ics (when visit my calendar) Can be used for Calendar Subscription
-  - Mac OS Immediate, Google Calendar and Outlook Calendar Not Yet Support Immediate
- showKanban Page Allow Reset Search Conditions
- Default Login Page Focus on the password textfield
2022-10-02
- Go Back Button Does not includes config->appUrl Bug Fixed
- 
Screenshot 2022-10-02 at 10 17 03 AM

- Solution to Delete Error Logged Hours e.g.45678 hours ( New Button )
- 
Screenshot 2022-10-02 at 12 01 00 PM
- Fix Bug Save and Close duplicate BASE URL. Remove BASE URL, SESSION lastPage contains it

### CODE REVIEW

File: config/configuration.sample.php [hunk 1]
```diff
@@ -4,17 +4,27 @@
 class config
 {
 	/* General */
-	public $sitename = "Leantime"; //Name of your site, can be changed later
+	public $sitename = "CALAW"; //Name of your site, can be changed later
```
Comment (marcelfolaron) [inline config/configuration.sample.php:7]: Could you remove the personal settings in this line and line 10+11

**Review by marcelfolaron — COMMENTED** [review]
This is great thank you! I will take that and make a couple improvements if you don't mind to stay in line with our current code set up.

File: config/configuration.sample.php (hunk 1 above)
Comment (calawhkbu) [reply config/configuration.sample.php]: removed the personal config  commit 268b5fe9992bd4edc9b47bf9fbe48d6da79d7fc0

**Comment by calawhkbu** [comment]
> This is great thank you! I will take that and make a couple improvements if you don't mind to stay in line with our current code set up.

Thanks for considering these features.
Yes please edit the code.

**Pull request closed without merging**